src/io/flutter/logging/FlutterLogTree.java

```diff
@@ -404,7 +415,7 @@ public void describeTo(@NotNull StringBuilder buffer) {
       buffer
         .append(TIMESTAMP_FORMAT.format(entry.getTimestamp()))
         .append(" ").append(entry.getSequenceNumber())
-        .append(" ").append(entry.getLevel())
+        .append(" ").append(FlutterLog.Level.forValue(entry.getLevel()).toDisplayString())
```

Comment:
I think we should display log level as text.
(Because user see in LogView is text eg `info/fine` instead of 800/400).

Reply:
👍

Reply:
One concern here is for custom user levels but we'll want to support those elsewhere too.  (Just food for thought!)

Reply:
I created a new method `getLevelName`.
So we can use it in the future.
https://github.com/flutter/flutter-intellij/pull/2455/commits/8ae8b519887e207fb4992c032095f71c337a9e9b